### PR TITLE
[feat]:history and Single Video Page

### PR DESCRIPTION
**Pull request by ultronjs**
-Added HIstory Feature
-added Single Video Page

### CODE REVIEW

File: src/components/Banner/Banner.jsx [hunk 1]
```diff
@@ -33,8 +35,18 @@ function Banner() {
...
+            Play
+          </button>
+          <button
+            className="banner_button"
+            onClick={() => navigate("/watchlater")}
+          >
+            My List
+          </button>
         </div>
         <h1 className="banner_description">{truncate(video?.overview, 150)}</h1>
```
Comment (sushil-mod) [inline src/components/Banner/Banner.jsx:51]: i like how you use turncate to show  ... if string length(overview) is maxed-out

File: src/components/RecommededVideosList.jsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+import { useParams } from 'react-router'
+import { useVideos } from '../context'
+import VideoCard from './VideoCard'
+
+function RecommededVideosList() {
+  const {videoId} = useParams()
+  const {videos} = useVideos()
+  const videoPlaying = videos.filter(video => video._id === videoId)
```
Comment (sushil-mod) [inline src/components/RecommededVideosList.jsx:11]: just a suggestion ,you can try .some() instead .filter as it returning only one video in array

**Review by sushil-mod — APPROVED** [review]
just a suggestion , Rest code is looks good

File: src/components/RecommededVideosList.jsx (hunk 2 above)
Comment (ultronjs) [reply src/components/RecommededVideosList.jsx:11]: Some return true or false.I want that video's data thatswhy filter

**Review by cjinc121 — APPROVED** [review]
liked use of separate components ,makes code more readable ,good to merge

**Comment by ultronjs** [comment]
Mmerging after 2 approvals

**Pull request merged**